### DISCUSSION TITLE
Permissions file not saving

### DISCUSSION THREAD

**Discussion by BorgoMadonno** [Kaltharos/RPGMods#4] [Q&A] [answered]
Hi, after I edit permissions file via ftp, I do reload with wetstone plugin but the file get back to the default. I tried restarting server too, same result.

Can you please check? It's not funny to make players able to use godmode commands..

**Comment by Kaltharos** [comment, marked as answer]
Generally, you should only edit any of the JSON files when your server is offline or they won't be saved.

However for this specific part, i have patched it in 0.1.4 and it will correctly load the new permission as long as your JSON structure is correct.
